src/js/node/tls.ts

```diff
@@ -252,7 +252,7 @@ function SecureContext(options) {
 }
 
 function createSecureContext(options) {
-  return new SecureContext(options);
```

Comment:
SecureContext is implemented in the lines above as
```ts
function SecureContext(options) {
  return new InternalSecureContext(options);
}
```

Reply:
`createSecureContext` should stay as-is and `SecureContext` should be

```ts
if (!(this instanceof SecureContext)) return new SecureContext(options);
InternalSecureContext.$call(this, options);
```

Reply:
Fixed in #d69631fc5d9a65b0d4f0197a2716b9081b1a6f9c